.github/actions/cleardown-tf-state/action.yaml

```diff
@@ -0,0 +1,12 @@
+name: "Cleardown terraform state"
```

Comment:
We tend to add 'action' to the end of the name to help distinguish it from workflows